src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery randomBall = new Lottery();
+        System.out.println(randomBall.getRandomBall());
+        System.out.println(randomBall.getRandomBall());
+        System.out.println(randomBall.getRandomBall());
```

Comment:
Let's print it using the loop. See [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class)